src/Spain/Mews.Fiscalizations.Spain/Converters/ModelToDtoConverter.cs

```diff
@@ -96,7 +96,7 @@ private SujetaType Convert(TaxSummary summary)
     {
         return new SujetaType
         {
-            Exenta = summary.TaxExempt.Map(items => items.Select(i => Convert(i)).ToArray()).GetOrNull(),
+            Exenta = summary.TaxExempt.Match(items => items.Select(i => Convert(i)).ToArray(), _ => null),
```

Comment:
Do we really need this? NoExenta just below uses Map so I would rather have the same behavior in both properties unless there is something else that I am missing? 😅

Reply:
Its because of the return type in the map/match, anyway i improved it further and got rid of the map and match.